### PR TITLE
Clarify page size 4KB.

### PR DESCRIPTION
**Pull request by jyao1**
Fix https://github.com/riscv-non-isa/riscv-ap-tee-io/issues/90
Fix https://github.com/riscv-non-isa/riscv-ap-tee-io/issues/91

### CODE REVIEW

File: specification/09-coveio_abi.adoc [hunk 1]
```diff
@@ -170,6 +170,10 @@ enable the created mappings until the TVM accepts the TDI in its TCB.
 
 The possible error codes returned in `sbiret.error` are shown below.
 
+`base_addr` value must be 4KB page aligned.
```
Comment (sameo) [inline specification/09-coveio_abi.adoc:173]: Should we just say `4KB-aligned`?

File: specification/09-coveio_abi.adoc [hunk 2]
```diff
@@ -170,6 +170,10 @@ enable the created mappings until the TVM accepts the TDI in its TCB.
...
+
+`num_pages` value is the number of 4KB pages.
```
Comment (sameo) [inline specification/09-coveio_abi.adoc:175]: I think we should change the argument here and use a `region_len` that should also be 4KB-aligned.

File: specification/09-coveio_abi.adoc [hunk 3]
```diff
@@ -320,7 +324,7 @@ first.
 
 `slot_id` must be between 0 and 7 inclusive.
 
-`cert_addr_out` must be page aligned.
+`cert_addr_out` must be 4KB page aligned.
```
Comment (sameo) [inline specification/09-coveio_abi.adoc:327]: `4KB-aligned`, and ditto for the other changes below.

File: specification/09-coveio_abi.adoc (hunk 1 above)
Comment (jyao1) [reply specification/09-coveio_abi.adoc]: yes, updated.

File: specification/09-coveio_abi.adoc (hunk 3 above)
Comment (jyao1) [reply specification/09-coveio_abi.adoc]: yes, updated.

File: specification/09-coveio_abi.adoc (hunk 2 above)
Comment (jyao1) [reply specification/09-coveio_abi.adoc]: yes, updated.

**Review by sameo — APPROVED** [review]

**Pull request merged**